src/main/java/frc/trigon/robot/commands/CommandConstants.java

```diff
@@ -1,18 +1,21 @@
-package frc.trigon.robot.constants;
+package frc.trigon.robot.commands;
 
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.util.Color;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import frc.trigon.robot.RobotContainer;
-import frc.trigon.robot.subsystems.ledstrip.LEDStripCommands;
-import frc.trigon.robot.subsystems.ledstrip.LEDStripConstants;
+import frc.trigon.robot.constants.OperatorConstants;
 import frc.trigon.robot.subsystems.swerve.SwerveCommands;
 import org.trigon.hardware.misc.XboxController;
+import org.trigon.hardware.misc.leds.LEDCommands;
+import org.trigon.hardware.misc.leds.LEDStrip;
 import org.trigon.utilities.mirrorable.MirrorablePose2d;
 
-import java.awt.*;
-
+/**
+ * A class that contains commands that run with the same values every usage.
+ */
 public class CommandConstants {
```

Comment:
1. explain the difference from the command factories
2. be more specific, what you wrote isn't totally understandable